Backend/app/rider-platform/rider-app/Main/spec/Storage/ticket.yaml

```diff
@@ -328,7 +393,12 @@ TicketBookingServiceCategory:
     name: Text
     bookedSeats: Int
     amount: Price
+    eventCancelledBy: Maybe CancelledBy
+    cancelledSeats: Maybe Int
+    amountToRefund: Maybe HighPrecMoney
     serviceCategoryId: Maybe Text
+    visitDate: Maybe Day
```

Comment:
Do we need visit Date in each table?

Reply:
actual yes, we are querying the respective tables to find entities from respecitive tables